pkg/metrics/nextgengrafana/tidb_summary.jsonnet

```diff
@@ -0,0 +1,167 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+local grafana = import 'grafonnet/grafana.libsonnet';
```

Comment:
Could we add a comment like: `Import Grafana components`

Reply:
Done